### PR TITLE
Making the Flycatcher useable

### PR DESCRIPTION
**Pull request by Amazinite**
#1314
Added a variant that is in the drone category for use by the player.

### CODE REVIEW

**Comment by LocalGod79** [comment]
i think you need to also clear the original version from the shipyard (or both variants will appear in the 'yard in their separate categories)

**Comment by Amazinite** [comment]
That didn't happen when I was testing it, but I'll do it just in case.

**Comment by KiLEdEnNis** [comment]
making ships that can carry interceptors (at least for the Wanderers) would fix it, and open more options, although those kinda carriers should be nearly defenseless to balance it
and even though no one would ever use Flycatchers, maybe they should be use-able, since there aren't shipyards in a few sectors where they appear

**Comment by Wrzlprnft** [comment]
Flycatchers are only interceptor to be able to actually appear in the systems, if i recall right.

File: data/wanderers.txt [hunk 1]
```diff
@@ -419,16 +419,32 @@ event "wanderers: first tech increase"
...
 		"Wanderer Anti-Missile"
 	outfitter "Wanderer Advanced"
 		"Wanderer Anti-Missile"
 		"Sunbeam Turret"
 		"Dual Sunbeam Turret"
+		
+#Reverse compatibility
+event "wanderers: flycatcher category"
+	clear "shipyard: Wanderer Basics"
+	clear "shipyard: Wanderer Advanced"
```
Comment (Disiuze) [inline data/wanderers.txt]: ditto

**Comment by Amazinite** [comment]
@Wrzlprnft Yes, that is the only reason they are interceptors, but it makes them completely useless to the player, hence why I made this pull to at least give Flycatchers some use to them.
@KiLEdEnNis I don't really see a need to make an entirely new type of ship that can carry interceptors when the fix is as easy as making the version sold in shipyards be in the drone category instead of the interceptor category. And drones/fighters are really nothing more than slightly smaller interceptors with no integrates fuel pods, meaning that they can't hyperdrive and need a host ship to do the system jumping for them. To make a ship that holds interceptors (which can already go to different systems on their own) kind of defeats the whole reason why fighters and drones are carried.

**Comment by Wrzlprnft** [comment]
My words we're aimed at the comment above mine.

I fully understand that PR and i like the idea :) the only point where the player can get a "workaround-flycatcher" would be through capturing. Angry wanderers don'r want you to use them, as it seems ;)

**Comment by Amazinite** [comment]
4ec09c9

**Pull request closed without merging**